cola/widgets/main.py

```diff
@@ -679,6 +694,16 @@ def export_state(self):
         show_status_filter = self.statuswidget.filter_widget.isVisible()
         state['show_status_filter'] = show_status_filter
         state['show_diff_line_numbers'] = self.diffeditor.show_line_numbers()
+
+        saved_toolbars = []
+        toolbars = self.findChildren(ColaToolBar)
+        for toolbar in toolbars:
+            saved_toolbars.append({'name': toolbar.windowTitle(),
+                                   'area': self.toolBarArea(toolbar),
+                                   'config': json.dumps(toolbar.get_config())})
```

Comment:
I think it would be better to not `json..dumps()` here.. this ends up with this in the final settings file:

```js
            "toolbar": [
                {
                    "config": "{\"data\": [{\"parent_id\": 122, \"child_id\": 40}, {\"parent_id\": 122, \"child_id\": 44}, {\"parent_id\": 123, \"child_id\": 60}, {\"parent_id\": 123, \"child_id\": 61}, {\"parent_id\": null, \"child_id\": 130}, {\"parent_id\": 121, \"child_id\": 20}, {\"parent_id\": 121, \"child_id\": 22}, {\"parent_id\": 121, \"child_id\": 21}], \"show_icons\": true}", 
                    "name": "Main", 
                    "area": 4
                }
            ]
```

note how there's embedded json inside that string.  It would be better to let this be the real raw value instead so that we don't have to compensate for it when reading and writing.  Down below we do `json.loads(...)` and we won't need to do that anymore.

If you make those changes you'll probably need to delete the toolbars from your settings, as it'll make the format incompatible for your current settings.